_docs/pql_queries.md

```diff
@@ -74,7 +74,7 @@ file { '/etc/puppetlabs/client-tools/puppetdb.conf':
 }
 ```
 
-## Contribute?!
+## Contribute?
```

Comment:
this is a stylistic choice and it's perfectly valid to express excitement in this way.